experiments/unbinned_spectrum_dataset/unbinned_datasets.py

```diff
@@ -0,0 +1,193 @@
+import numpy as np
+from astropy.io import fits
+from gammapy.spectrum import (
+    SpectrumDatasetMaker,
+    SpectrumDatasetOnOff,
+    ReflectedRegionsBackgroundMaker,
+)
+from gammapy.utils.regions import list_to_compound_region
+
+
+class UnbinnedSpectrumDatasetOnOff(SpectrumDatasetOnOff):
+    """The UnbinnedSpectrumDatasetOnOff has the events and events_off 
+    attritubutes compared to the SpectrumDatasetOnOff from which it inherits."""
+
+    def __init__(
```

Comment:
This you can further simplify by passing on the parameters to the `SpectrumDatasetOnOff` like so:

```python3

def __init__(self, events, events_off, **kwargs):
    self.events = events
    self.events_off = events_off
    super().__init__(**kwargs)
```

Reply:
Many thanks, done!